webstruct/tests/test_webannotator.py

```diff
@@ -91,15 +256,15 @@ def test_wa_convert(self):
             <p>
               __START_ORG__ Scrapinghub
                 <b>Inc __END_ORG__  has</b>an
-                <b>__START_CITY__office</b>in Montevideo __END_CITY__   __START_ORG__ SH __END_ORG__
+                <b> __START_CITY__ office</b>in Montevideo __END_CITY__   __START_ORG__ SH __END_ORG__ cool
```

Comment:
Why is space added before start token?

Reply:
Just for strictness. In html tokenizer we use ' __START/END_SMTH__ '(with spaces) tags, so in towebannotator function we should look for tags with spaces.